### DISCUSSION TITLE
blog/clickbait/

### DISCUSSION THREAD

**Discussion by giscus** [yandili/yandi.space#1] [General]
# blog/clickbait/

This is a description of the site.

http://10.185.9.48:9112/blog/clickbait/

**Comment by yandili** [comment]
测试一下评论系统，看看支持哪些功能，
- 是否能成功删除
- 是否能回复

**Reply by yandili** [reply]
测试一下回复功能

**Reply by yandili** [reply]
可以在github的Discussions进行删除，还算方便
